engine/items/item_effects.asm

```diff
@@ -499,11 +499,11 @@ PokeBallEffect:
 	ld a, [wBattleType]
 	cp BATTLETYPE_CONTEST
 	jmp z, .catch_bug_contest_mon
-	cp BATTLETYPE_LEGENDARY
-	jr nz, .not_celebi ; false positive for other legendaries, but that's okay
+
+	; Set wBattleResult bit 6 to record
+	; that the pokémon was caught.
```

Comment:
`set BATTLERESULT_CAUGHT_POKEMON, [hl]` is obvious enough that this comment can be removed. Especially since, if the value of `BATTLERESULT_CAUGHT_POKEMON` were ever changed from 6, the comment would become misleading.